resources/factions/Syria-Lebanon_2005_Allied_Sword.json

```diff
@@ -0,0 +1,102 @@
+{
+  "country": "Combined Joint Task Forces Red",
+  "name": "Syria-Lebanon 2005 (Allied Sword)",
+  "authors": "Fuzzle",
+  "description": "<p>Syria-Lebanon alliance in a modern setting with several imported Russian assets. Designed for use with the Allied Sword scenario.</p>",
+  "aircrafts": [
+    "MiG_23MLD",
+    "MiG_25PD",
+    "MiG_29S",
+    "Su_17M4",
+    "Su_24M",
+    "Su_30",
+    "Su_34",
+    "L_39ZA",
+    "Tu_22M3",
+    "Mi_24V",
+    "Mi_8MT",
+    "SA342M",
+    "SA342L"
+  ],
+  "awacs": [
+    "A_50"
+  ],
+  "tankers": [
+    "IL_78M"
+  ],
+  "frontline_units": [
+    "IFV_BMP_1",
+    "IFV_BMP_2",
+    "APC_BTR_80",
+    "IFV_BRDM_2",
+    "APC_MTLB",
+    "MBT_T_55",
+    "MBT_T_72B",
+    "MBT_T_90",
+    "SPAAA_ZSU_57_2"
+  ],
+  "artillery_units": [
+    "MLRS_BM_21_Grad_122mm",
+    "SPH_2S1_Gvozdika_122mm"
+  ],
+  "logistics_units": [
+    "Truck_Ural_375",
+    "LUV_UAZ_469_Jeep"
+  ],
+  "infantry_units": [
+    "Paratrooper_AKS",
+    "Infantry_AK_74_Rus",
+    "Paratrooper_RPG_16",
+    "MANPADS_SA_18_Igla_S_Grouse"
+  ],
```

Comment:
Most if not all of these names are wrong. I don't think any of the units will show up in the buy menu?

Reply:
sorry your correct remove that file and i'll update the bug. I forgot about the faction changes when I checked this one out

Reply:
Okay, in that case I'm going to close this PR. It's easier for me to just submit changes from the bug tracker than it is for me to revert half of a PR.